.github/workflows/check-catalyst.yaml

```diff
@@ -32,7 +32,8 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get -y -q install ninja-build make cmake clang libomp-dev
+          sudo apt-get -y -q install ninja-build make cmake clang libomp-dev python3 python3-pip
```

Comment:
Mmm... that is weird because we are installing python3 here: https://github.com/PennyLaneAI/catalyst/pull/540/files#diff-7cddfc77dfe24b2596ab45690b5d0b7b7881d61fb5fe7ba0beaa2fec6cb62ddfR107-R110 I will delete this and try, but I believe that the reason I added this is because it failed to compile in the past.

Reply:
let's give it a try! maybe you're right but it's weird they already mention the support in `ubuntu-latest` docs.

Reply:
Got rid of them here https://github.com/PennyLaneAI/catalyst/pull/540/commits/e3c81e8408c2a3b46f4d0296162857a9c9f1a168 and here https://github.com/PennyLaneAI/catalyst/pull/540/commits/a650111fbc78f66077f49284b01b9ad544c2ffa4 . Thanks!